app/Model/Candidate/CandidateFunction.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Model\Candidate;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity()
+ * @ORM\Table()
+ */
+class CandidateFunction
```

Comment:
Nemůže tohle být enum? Potřebujeme to mít v databázi?

Reply:
Přemýšlel jsem nad tím, ale reálně potřebuju ke kazdé funkci více údajů než jen název, takze jsem to hodil do db.

Reply:
I tak. Nechci nutit přímo implementaci jako enum, pokud je to blbej fit v kombinaci větším množstvím údajů, ale minimálně nějakou statickou definici (třeba i jen "hloupou" implementaci té repository s reprezentací polem).
Mně se na tomto přístupu obecně nelíbí to, že pro správné fungování musí být nějaký konkrétní stav databáze, kterého však nelze dosáhnout jinak než nějakým iniciálním nalitím dat.

Pokud se to nikdy nemění a nejsou toho desítky, tak je IMO vždy lepší to mít natvrdo někde zanesený. Vím, že by mě spousta milovníků DB normalizace hnalo, ale ve výsledku je to o dost praktičtější :grin: